<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Send a Message</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../usr/local/share/ksms/icon.png</normaloff>../../../../usr/local/share/ksms/icon.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QComboBox" name="deviceSelect">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>311</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Select a Device</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="destinationSelect">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>221</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Type a Phone Number</string>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentText">
    <string extracomment="Phone Number"/>
   </property>
   <property name="maxVisibleItems">
    <number>14</number>
   </property>
  </widget>
  <widget class="QTextEdit" name="message">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>311</width>
     <height>141</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Enter Message</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>311</width>
     <height>34</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Send Message</string>
   </property>
   <property name="text">
    <string>Send Message</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>311</width>
     <height>34</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Check for Available Devices</string>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="destinationButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <underline>false</underline>
    </font>
   </property>
   <property name="toolTip">
    <string>Open Contacts</string>
   </property>
   <property name="styleSheet">
    <string notr="true">Text-align:left</string>
   </property>
   <property name="text">
    <string>Destination</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>refreshButton</zorder>
  <zorder>deviceSelect</zorder>
  <zorder>destinationSelect</zorder>
  <zorder>message</zorder>
  <zorder>sendButton</zorder>
  <zorder>destinationButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
